src/PowerSystems.jl

```diff
@@ -180,6 +180,7 @@ export VariableReserveNonSpinning
 export PTDF
 export Ybus
 export LODF
+export Adjacency
```

Comment:
```suggestion
export AdjacencyMatrix
```

Reply:
It's a `<:PowerNetworkMatrix`, so I'm inclined to keep it as `Adjacency`